packages/lesswrong/components/sunshineDashboard/SunshineNewUsersInfo.tsx

```diff
@@ -140,6 +141,22 @@ const styles = (theme: ThemeType): JssStyles => ({
     },
   },
 })
+
+export interface UserContentCountPartial {
+  postCount?: number,
+  commentCount?: number
+}
+
+export function getCurrentContentCount(user: UserContentCountPartial) {
+  const postCount = user.postCount || 0
+  const commentCount = user.commentCount || 0
+  return postCount + commentCount;
+}
+
+export function getNewSnoozeUntilContentCount(user: UserContentCountPartial, contentCount: any) {
```

Comment:
`contentCount: number`?

Reply:
nah

Reply:
?

Reply:
I... do not remember why I thought it was appropriate to reply "nah" (it's not even a particularly good joke), but apparently I already updated it to use "number." Wuckles.